en/config_mc/pid_tuning_guide_multicopter.md

```diff
@@ -49,14 +51,59 @@ Here are some general points to follow when tuning:
-- Yaw rate control ([MC_YAWRATE_P](../advanced_config/parameter_reference.md#MC_YAWRATE_P), [MC_YAWRATE_I](../advanced_config/parameter_reference.md#MC_YAWRATE_I), [MC_YAWRATE_D](../advanced_config/parameter_reference.md#MC_YAWRATE_D))
+> **Note** A well-tuned rate controller is very important as it affects *all* flight modes.
+  A badly tuned rate controller will be visible in [Position mode](../flight_modes/position_mc.md), for example, as "twitches" (the vehicle will not hold perfectly still in the air).
 
+#### Rate Controller Architecture/Form
 
-> **Note** A well-tuned rate controller is very important as it affects *all* flight modes. A badly tuned rate controller will be visible in [Position mode](../flight_modes/position_mc.md), for example, as "twitches" (the vehicle will not hold perfectly still in the air).
+PX4 supports two (mathematically equivalent) forms of the PID rate controller in a single "mixed" implementation: [Parallel](#parallel_form) and [Standard](#standard_form).
+
+Users can select the form that is used by setting the proportional gain for the other form to "1" (i.e. in the diagram below set **K** to 1 for the parallel form, or **P** to 1 for the standard form - this will replace either the K or P blocks with a line).
+
+![PID_Mixed](../../images/mc_pid_tuning/PID_algorithm_Mixed.png)
+<!-- The drawing is on draw.io: https://drive.google.com/file/d/1hXnAJVRyqNAdcreqNa5W4PQFkYnzwgOO/view?usp=sharing -->
+- _G(s)_ represents the angular rates dynamics of a vehicle
+- _r_ is the rate setpoint
+- _y_ is the body angular rate (measured by a gyro)
+- _e_ is the error between the rate setpoint and the measured rate
+- _u_ is the output of the PID controller
+
+
+The two forms are described below.
+
+
+> **Note** The derivative term (**D**) is on the feedback path in order to avoid an effect known as the [derivative kick](http://brettbeauregard.com/blog/2011/04/improving-the-beginner%E2%80%99s-pid-derivative-kick/).
+
+<span></span>
+> **Tip** For more information see:
+  - [Not all PID controllers are the same](https://www.controleng.com/articles/not-all-pid-controllers-are-the-same/) (www.controleng.com) 
+  - [PID controller > Standard versus parallel (ideal) PID form](https://en.wikipedia.org/wiki/PID_controller#Standard_versus_parallel_(ideal)_PID_form) (Wikipedia)
+
+
+##### Parallel Form {#parallel_form}
+
+The *parallel form* is the simplest form, and is (hence) commonly used in textbooks.
+In this case the output of the controller is simply the sum of the proportional, integral and derivative actions.
+
+![PID_Parallel](../../images/mc_pid_tuning/PID_algorithm_Parallel.png)
+
+##### Standard Form {#standard_form}
+
+The *standard form* is mathematically equivalent to the *parallel form* but facilitates manual tuning as the K gain represents the global "aggressiveness" of the controller. 
```

Comment:
Why this is a good think isn't clear to me - i.e. why should user try this one or the other one?

In your original post you said something like "facilitate the tuning of a new airframe as it decouples properly the integral and derivative time constants. A user should be able to take the gains of a similar platform (size/inertia) and simply adjust this gain to have it flying properly.
Until now, with the pure parallel form, if the thrust to weight ratio is different from the reference config, the tuning has to be done almost from scratch."

Is that true? Does this make tuning easier in other ways? To me the aggressiveness etc doesn't mean all that much.

Reply:
Ok, I'll reincorporate that explanation then. Just for you to know, I see no advantage in the parallel form. Most likely, this became the default because it came from a student project.

Reply:
Cool. The main thing is to give people direction - which will be easier? When should they use one over the other? Why did we choose to give them two choices?